packages/lib/src/components/Profile/components/ProfileDataCard/components/PasswordChangeForm.tsx

```diff
@@ -21,31 +21,75 @@
 
 import {
   PasswordChange,
-  passwordChangeSchema,
+  passwordChangeSchemaAdmin,
+  passwordChangeSchemaUser,
+  PasswordStrengthRules,
 } from '@frachtwerk/essencium-types'
-import { Box, Button, Flex, PasswordInput, Stack, Text } from '@mantine/core'
+import {
+  Box,
+  Button,
+  Flex,
+  PasswordInput,
+  Popover,
+  Stack,
+  Text,
+} from '@mantine/core'
 import { useTranslation } from 'next-i18next'
+import { useState } from 'react'
 import { Controller } from 'react-hook-form'
 
 import { useZodForm } from '../../../../../hooks'
 import classes from './PasswordChangeForm.module.css'
+import { PasswordRequirement } from './PasswordRequirement'
 
 type Props = {
   handlePasswordUpdate: (
     oldPassword: PasswordChange['password'],
     newPassword: PasswordChange['password'],
   ) => void
   isLoading: boolean
+  isAdmin: boolean
```

Comment:
This could be optional without problems and makes the component a bit more flexibel.